README.md

```diff
@@ -2,7 +2,9 @@
 
 A nova field for Laraberg
 
-## Installation
+## Quick start
+
+### Installation
 
 Install via composer
```

Comment:
`Composer` graag met hoofdletter omdat het een naam is, behalve in code uiteraard ;-)

Reply:
En tussen regel en code ook graag een `;`.